1-js/02-first-steps/06-alert-prompt-confirm/article.md

```diff
@@ -1,105 +1,111 @@
-# Interaction: alert, prompt, confirm
+# Interação: alert, prompt, confirm
 
-As we'll be using the browser as our demo environment, let's see a couple of functions to interact with the user: `alert`, `prompt` and `confirm`.
+Esta parte do tutorial tem como objetivo cobrir o JavaScript "como é ", sem ajustes específicos de ambiente.
```

Comment:
Could you update your translation with respect to the english version? https://github.com/javascript-tutorial/en.javascript.info/blob/master/1-js/02-first-steps/06-alert-prompt-confirm/article.md some parts have changed. Thank you!